codemon/CodemonReg.py

```diff
@@ -6,22 +6,31 @@
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 
 def codemonReg():
-    print(ROOT_DIR)
-    path = os.path.join(ROOT_DIR, "codemon")
+    #print(ROOT_DIR)
+    prefix = '.' if os.name != 'nt' else ''
+    file_name = prefix + "codemon"
+    path = os.path.join(ROOT_DIR, file_name)
```

Comment:
Please remove `#print(ROOT_DIR)`. This method of finding prefix is not proper and can lead to bugs in future. Read code in other files and follow that code style. Also, the file is not created at linux root directory. For example,
```
anking@LAPTOP-GLS2PDLH:~$ codemon reg
/home/anking/.local/lib/python3.6/site-packages/codemon
Codemon User Registration

Please enter the following data. Your sensitive information is collected for making your life easier while using Codemon and will be stored on your local machine. It won't be stored or used by Codemon developers for any purpose.
```
See that the ROOT_DIR is somewhere else inside my python distribution. For my machine, `/home/anking` is the root directory. The .codemon folder should be created there and the resulting path would be 
```
/home/anking/.codemon/meta.json  
```